### ISSUE TITLE
Sites: provide My Blogs replacement for editing site visibility

### ISSUE THREAD

**Issue by lancewillett** [Automattic/wp-calypso#8335]
Provide a way to edit the visibility of WordPress.com and Jetpack sites within Calypso. Currently this is only available in wp-admin.

Example from `testerlance` user, from the My Blogs global dashboard:

URL: https://dashboard.wordpress.com/wp-admin/index.php?page=my-blogs

![screen shot 2016-09-29 at 11 45 46](https://cloud.githubusercontent.com/assets/66797/18967516/592aa38a-863a-11e6-98e6-af83fde427c2.png)
# calypso-gaps

**Comment by mtias** [comment]
This is not necessarily something we want to build in Calypso — it's proven to be a confusing feature, where users mistake "hiding" a site for making it private, and we think by having a solid site picker experience (search, recent sites, more grouping features for networks) we may not need the hide feature at all.

**Comment by folletto** [comment]
This is definitely a power user feature. I'd still aim to have it _in some form_ in Calypso (where it could be the same UI to manage networks/grouping, etc), but should be accessible in a marginal UI. I think the addition at the bottom of the sites picker is effective for that reason, we just need to build the page with the list.

I agree on a better label however. "Hidden" is very confusing.

---

Screenshot for reference of the current link to access the feature, even if it currently points to wp-admin:

![screen shot 2016-10-29 at 20 04 47](https://cloud.githubusercontent.com/assets/4389/19832108/f946c110-9e12-11e6-9a96-6c9bdef0bef4.png)

**Comment by lancewillett** [comment]
Next steps: let's remove all the links to the My Blogs global dashboard from Calypso, and go with the smart defaults of ordering the sites.

CC @mtias

**Comment by lancewillett** [comment]
At the very least, let's remove the wp-admin link there, and leave the text something like:

```You have x sites hidden.
Use search to access them.```

**Comment by lancewillett** [comment]
@mtias  PR to remove link is ready for review: #9587

**Comment by folletto** [comment]
_(Discussion moved to the PR)_

**Comment by lancewillett** [comment]
A few key notes from data research:

1. Fewer than than 0.3% of WordPress.com user accounts have hidden blogs (internal ref: p10gg3-3dm-p2).
2. From a study we did in July 2015 (internal reference: p10gg3-3om-p2) looking at a specific cohort of 105K sites on WordPress.com, we found that 95% of all the users have only one blog, 4.4% have two blogs. The overall percentage of users with two blogs or more is thus 5%.
